include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 14
+#define SOF_ABI_MINOR 15
```

Comment:
@xiulipan This is not right. Debug API is 14 is classifier and merged FW patch https://github.com/thesofproject/sof/pull/2398/commits

Reply:
@lgirdwood can you clarify if we will fix to use 3 ABI version for these patches?

Reply:
We'll go with 14